### PR TITLE
[WIP] 181: Bold labels

### LINKED ISSUE #181: Proposed Rezoning label should be bolded on the area map
When a user creates or new district or changes the label for an existing district, those new labels should be bolded. This requires updates to the label styling and the legend.

### PR DESCRIPTION
**Pull request by trbmcginnis**
This PR uses data-driven styling to "bold" the labels of zoning districts edited by users. The styling method is similar to how we dynamically style `{label}` for the `text-field` property in `underlyingZoningDistrictsLabelsLayer`.

Match expressions were used in the labels layer for both the area map (`project-geom-layers.js`) and for the editing maps (`underlying-zoning.js`). 

Because mapbox does not currently support data-driven styling for the `text-font` property, a workaround was implemented by editing the `text-letter-spacing`, `text-halo-color`, `text-halo-width`, and `text-halo-blur` properties. 

Closes #181

### CODE REVIEW

**Comment by trbmcginnis** [comment]
bold_app_maps

**Comment by trbmcginnis** [comment]
bold_app_maps_2

File: app/components/project-geometries/modes/draw/feature-label-form.js [hunk 1]
```diff
@@ -21,10 +21,32 @@ export default class FeatureLabelFormComponent extends Component {
...
+  textHaloBlur;
+
   @action
   handleSelectChange(newLabel) {
     this.set('label', newLabel);
-    this.updateSelectedFeature(newLabel);
+    this.set('textLetterSpacing', 0.2);
+    this.set('textHaloColor', '#444');
+    this.set('textHaloWidth', 1);
+    this.set('textHaloBlur', 0);
```
Comment (allthesignals) [inline app/components/project-geometries/modes/draw/feature-label-form.js]: We should pull these out into constants above the class definition:

```
const DEFAULT_TEXT_LETTER_SPACING = 0.2;
```

Etc

**Comment by allthesignals** [comment]
Let's get the conflict resolved and merged

**Pull request closed without merging**